elyra/pipeline/airflow/package_catalog_connector/airflow-package-catalog.json

```diff
@@ -56,11 +56,30 @@
         },
         "airflow_package_download_url": {
           "title": "Airflow package download URL",
-          "description": "URL where the Apache Airflow package wheel can be downloaded.",
+          "description": "URL where the Apache Airflow package wheel can be downloaded",
           "type": "string",
           "uihints": {
```

Comment:
I think we should add a URI format specifier here so validation can ensure a valid format...
```suggestion
          "type": "string",
          "format": "uri",
          "uihints": {
```